index.js

```diff
@@ -123,62 +103,23 @@ function YahooFantasy(consumerKey, consumerSecret) {
   // this.users.fetch = _.bind(this.users.fetch, this);
 }
 
-YahooFantasy.prototype.setUserToken = function(userToken, userSecret, userSession) {
-  this.yuser.token = userToken;
-  this.yuser.secret = userSecret;
-  this.yuser.sessionHandle = userSession;
-};
-
-YahooFantasy.prototype.refreshUserToken = function() {
-  var deferred = Q.defer();
-  var self = this;
-
-  var now = Math.floor(new Date().getTime() / 1000);
-  var refresh_data = querystring.stringify({
-    oauth_nonce: now.toString(36),
-    oauth_consumer_key: self.consumer.key,
-    oauth_signature_method: 'plaintext',
-    oauth_signature: self.consumer.secret + '&' + self.yuser.secret,
-    oauth_version: '1.0',
-    oauth_token: self.yuser.token,
-    oauth_timestamp: now,
-    oauth_session_handle: self.yuser.sessionHandle
-  });
-
-  https.get('https://api.login.yahoo.com/oauth/v2/get_token?' + refresh_data, function(res) {
-    var s = '';
-    res.on('data', function(d) {
-      s += d;
-    });
-
-    res.on('end', function() {
-      var data = querystring.parse(s);
-      self.setUserToken(data.oauth_token, data.oauth_token_secret, data.oauth_session_handle);
-
-      return deferred.resolve();
-    });
-  });
-
-  return deferred.promise;
+YahooFantasy.prototype.setUserToken = function(userToken) {
```

Comment:
from what I understand, you're suggesting we just remove the OAuth stuff completely from the library, and the user will authenticate externally, and will just provide the token to the API?